frontend/src/common/components/BadgeContact/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { HiOutlineArrowRight } from 'react-icons/hi';
+
+import Button from '../Button';
+import Container from '../Container';
+
+import {
+  BadgeListContainer,
+  BadgeList,
+  Badge,
+  BadgeIcon,
+  BadgeName,
+  Description,
+} from './styled';
+
+const BadgeContact: React.FC = () => {
```

Comment:
should rename to something like `BadgeContacts` <- wthi `s`   or `BadgeContactSection`